README.md

```diff
@@ -100,6 +100,9 @@ listAdapter.convertToString(person) // Produces JSON API String from a Person li
 listAdapter.convertFromString(inputJsonString) // Produces Person list from JSON API String
 ```
 
+In some cases, a need for custom modification of certain aspects of model classes is preferred. With this in mind, we've enabled the option for clients to customize 
+JsonApiModel if such a need occurs. Clients are able to set optional params on a model class such as links or meta(explained in its own chapter), but also to customize the default `id` of a model. Clients can simply set/get `id` on a JsonApiModel depending on the case at hand.
+
```

Comment:
There is a whole section related to `JsonApiModel`. I would move this there. Also use the ticks around "JsonApiModel" to make it look like code -> `JsonApiModel`

Reply:
As you wish, I think it's better to leave it here on customization part but will move it 👍